cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -668,12 +668,12 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
 
         getGroupAccessData: function() {
             var userPartitionId = this.getSelectedEnrollmentTrackId(),
-                groupAccess = {};
-            if (userPartitionId !== 'none' && !isNaN(userPartitionId)) { // If the selected partition ID is not none
-                groupAccess[userPartitionId] = this.getSelectedCheckboxesByDivId(userPartitionId);
-                return groupAccess;
+                group_access = {};
```

Comment:
JS style requires us to use camelCase, so we'll need to change this back or else our linter will fail.